src/main/java/com/google/step/datamanager/RestaurantManager.java

```diff
@@ -0,0 +1,14 @@
+package com.google.step.datamanager;
+
+import com.google.step.model.Restaurant;
+
+public interface RestaurantManager {
```

Comment:
Add some comments?

Reply:
Ok, I added in the RestaurantManagerDatastore